runtime/doc/starting.txt

```diff
@@ -545,13 +545,13 @@ accordingly.  Vim proceeds in this order:
 	":version" command.  Mostly it's "$VIM/vimrc".
 	For the Macintosh the $VIMRUNTIME/macmap.vim is read.
 
-	  *VIMINIT* *.vimrc* *_vimrc* *EXINIT* *.exrc* *_exrc* *$MYVIMRC*
+	  *NVIMINIT* *.vimrc* *_vimrc* *EXINIT* *.exrc* *_exrc* *$MYVIMRC*
```

Comment:
Well there's plenty more `.vimrc` references than just those pertaining to this tag, and it's not directly related to this PR so I don't think it should be done here.